packages/docs/content/components/tooltip.mdx

```diff
@@ -1,12 +1,23 @@
 ---
 title: Tooltip
 status: draft
+intro: Tooltips provide additional information upon hover, focus or click. The information should be contextual, useful, and nonessential information.
+core:
+  githubLink: design-system/src/components/Tooltip
+  sketchLink: B8CDD372-C06C-4C5F-9D08-3E78B5AD6CC2
+  storybookLink: components-tooltip--icon-trigger
+healthcare:
+  githubLink: design-system/src/components/Tooltip
+  sketchLink: 6B9E0E70-93E2-4068-B7A5-317FADC175AE
+  storybookLink: components-tooltip--icon-trigger
+medicare:
+  githubLink: design-system/src/components/Tooltip
+  sketchLink: 9D306E81-ED90-4FA3-B298-D7D44F4405AF
+  storybookLink: components-tooltip--icon-trigger
```

Comment:
What do you think of this structure. In yaml, it is equivalent to an object of key/value pairs.

The idea is that every component page would have the three 'core' values defined + the sketch links for healthcare and medicare. There are a few components that are brand-specific, so they may need different github or storybook urls than core.

Similar to other places in our system, it's a bit like a cascade, where if a medicare storybook link is not defined, the core one will be used.

Thoughts?

Reply:
I was going to ask about a cascade option - since the GH and SB links are the same for all three it looked like a value was required

Reply:
I did that for this PR, but it is confusing. I'll update.
I also talked to Scott and sketch links will not cascade, but I think the other two can (for now)